spec/acceptance/init_spec.rb

```diff
@@ -2,73 +2,72 @@
-      expect(result[0]['result']['status']).to eq('up to date')
-      expect(result[0]['result']['version']).to match(%r{\d+\.\d+\.\d+})
+      result = run_bolt_task('package', 'action' => 'status', 'name' => 'pry', 'provider' => 'puppet_gem')
+      expect(result.exit_code).to eq(0)
+      expect(result.stdout).to match(%r{up to date|install ok installed})
     end
   end
 
   describe 'install without puppet' do
     it 'installs rsyslog', unless: (operating_system_fact == 'windows') || redhat_six do
-      result = run_task('package', 'default', 'action' => 'install', 'name' => 'rsyslog')
-      expect(result[0]['status']).to eq('success')
-      expect(result[0]['result']['status']).to match(%r{install|present})
+      result = run_bolt_task('package', 'action' => 'install', 'name' => 'rsyslog')
+      expect(result.exit_code).to eq(0)
+      expect(result.stdout).to match(%r{install|present})
     end
   end
 
   describe 'uninstall' do
     before(:all) do
-      apply_manifest_on(default, 'package { "pry": ensure => "present", provider => "puppet_gem", }')
+      apply_manifest('package { "pry": ensure => "present", provider => "puppet_gem", }')
     end
 
     it 'uninstalls pry' do
-      result = run_task('package', 'default', 'action' => 'uninstall', 'name' => 'pry', 'provider' => 'puppet_gem')
-      expect(result[0]['status']).to eq('success')
-      expect(result[0]['result']['status']).to eq('uninstalled')
+      result = run_bolt_task('package', 'action' => 'uninstall', 'name' => 'pry', 'provider' => 'puppet_gem')
+      expect(result.exit_code).to eq(0)
+      expect(result.stdout).to eq('uninstalled')
     end
+
     it 'status' do
-      result = run_task('package', 'default', 'action' => 'status', 'name' => 'pry', 'provider' => 'puppet_gem')
-      expect(result[0]['status']).to eq('success')
-      expect(result[0]['result']['status']).to eq('absent')
+      result = run_bolt_task('package', 'action' => 'status', 'name' => 'pry', 'provider' => 'puppet_gem')
+      expect(result.exit_code).to eq(0)
+      expect(result.stdout).to match(%r{absent|uninstalled})
     end
   end
 
   describe 'upgrade', if: (operating_system_fact == 'CentOS' && os[:release].to_i == 7) do
     before(:all) do
-      apply_manifest_on(default, 'package { "httpd": ensure => "present", }')
+      apply_manifest('package { "httpd": ensure => "present", }')
     end
 
     it 'upgrade httpd' do
-      result = run_task('package', 'default', 'action' => 'upgrade', 'name' => 'httpd')
-      expect(result[0]['status']).to eq('success')
+      result = run_bolt_task('package', 'action' => 'upgrade', 'name' => 'httpd')
+      expect(result.exit_code).to eq('success')
       expect(result[0]['result']['version']).to match(%r{2.4.6-\d+.el7.centos})
       expect(result[0]['result']['old_version']).to match(%r{2.4.6-\d+.el7.centos})
+      inventory_hash = remove_feature_from_group(inventory_hash, 'puppet-agent', 'ssh_nodes')
```

Comment:
Would this not be better in an `after(:all)` at the top context?
https://github.com/puppetlabs/puppetlabs-package/blob/bcba5490f13a10df82c56c390c78fe059f146a8f/spec/acceptance/init_spec.rb#L67-L70